README.md

```diff
@@ -25,9 +25,19 @@ Going forward, all images shall be available at `quay.io/oauth2-proxy/oauth2-pro
 
     a. Download [Prebuilt Binary](https://github.com/oauth2-proxy/oauth2-proxy/releases) (current release is `v6.1.1`)
 
-    b. Build with `$ go get github.com/oauth2-proxy/oauth2-proxy` which will put the binary in `$GOROOT/bin`
-
-    c. Using the prebuilt docker image [quay.io/oauth2-proxy/oauth2-proxy](https://quay.io/oauth2-proxy/oauth2-proxy) (AMD64, ARMv6 and ARM64 tags available)
+    b. Using the prebuilt docker image quay.io/oauth2-proxy/oauth2-proxy (AMD64, ARMv6 and ARM64 tags available)
+
+    c. Clone source code and build binary. Make sure you install golang [Instructions Here](https://golang.org/doc/install) and ensure that the $GOPATH environment variable is set correctly [Instructions Here](https://golang.org/doc/gopath_code.html). 
+
+    Then perform the following commands:
+    ```
+    $ mkdir -p $GOPATH/src/github.com/oauth2-proxy
+    $ cd $GOPATH/src/github.com/oauth2-proxy
+    $ git clone https://github.com/oauth2-proxy/oauth2-proxy
+    $ cd oauth2-proxy
```

Comment:
All of this seems too verbose a little redundant to the `Clone source code...` instruction I think.

I also don't think any of the old enforced path structure and $GOPATH are required anymore (was this 1.12? I can't recall).

For instance, I can build from my `~/src/oauth2-proxy` directory I cloned the repo into as long as I set my golang version I manage with `asdf` to `1.15` for that directory.

So I lean against prescribed conventions that aren't mandatory and let the developer install this repo in a way that they are comfortable in for their golang development environment.

Reply:
Could just sum it up and say clone repo and run make build?

Reply:
That might be best, I think brevity is best in this area.

Long term I think more sections needed to be added to this README to summarize key areas expanded upon in the Docs (We can for instance add more detailed build isntructions to the Docs).  And in that vision, I think this is only applicable to a very small subset of potential users so I don't want to waste too much page space real estate on it.

Reply:
Alrighty I made it as discussed.